core/src/main/java/org/apache/carbondata/core/metadata/schema/table/ColumnTableRelation.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.core.metadata.schema.table;
+
+public class ColumnTableRelation {
+  private String parentColumnName;
+  private String parentColumnId;
+  private String parentTableName;
+  private String parentDatabaseName;
+  private String parentTableId;
+
+  public ColumnTableRelation(String parentColumnName, String parentColumnId, String parentTableName,
```

Comment:
what is the difference with `ParentColumnTableRelation` class? why don't you use here

Reply:
This class is moved directly from `case class ColumnTableRelation` in carbonTableSchemaCommon.scala in master
I can change two member variables to `ParentColumnTableRelation`, but still not very intuitive. Any suggestion?